ext/MathOptAILuxExt.jl

```diff
@@ -211,7 +211,7 @@ end
 
 function _add_predictor(
     predictor::MathOptAI.Pipeline,
-    ::Lux.WrappedFunction{:direct_call,typeof(Lux.softmax)},
+    ::Lux.WrappedFunction{typeof(Lux.softmax)},
```

Comment:
Changed in Lux@1.0.0: https://github.com/LuxDL/Lux.jl/commit/66c9f47d99b91e8f7e89a936dc04ef6aef0fabe7